grammars/go.cson

```diff
@@ -128,6 +128,10 @@
     'match': '\\b(iota|true|false|nil)\\b'
   }
   {
+    'match': '\\;'
+    'name': 'punctuation.terminator.statement.go'
```

Comment:
`punctuation` is not a known root group used by TextMate (https://manual.macromates.com/en/language_grammars#naming_conventions). As we try to follow the TextMate guidelines to make packages easily portable between TextMate and Atom, the correct scope selector for this one should be `keyword.operator.go` or maybe `keyword.operator.terminator.go` if you would like to specifically scope the semicolon.

Personally I think the current `keyword.operator.go` is the better choice looking at the language specs https://golang.org/ref/spec#Operators_and_Delimiters but `keyword.operator.terminator.go` would be acceptable as well.

While looking at it, I noticed the semicolon is actually already scoped correctly here https://github.com/atom/language-go/blob/master/grammars/go.cson#L162 so by adding this you actually try to scope the same operator twice...